lib/cudadrv/profile.jl

```diff
@@ -25,26 +25,37 @@ module Profile
 
 using ..CUDA
 
+function find_nsys()
+    if haskey(ENV, "JULIA_CUDA_NSYS")
+        return ENV["JULIA_CUDA_NSYS"]
+    elseif haskey(ENV, "_") && contains(ENV["_"], r"nsys"i)
+        # NOTE: if running as e.g. Jupyter -> nsys -> Julia, _ is `jupyter`
+        return ENV["_"]
+    else
+        # look at a couple of environment variables that may point to NSight
+        nsight = nothing
+        for var in ("LD_PRELOAD", "CUDA_INJECTION64_PATH", "NVTX_INJECTION64_PATH")
+            haskey(ENV, var) || continue
+            for val in split(ENV[var], ':')
```

Comment:
The delimiter on Windows would be `;`, not `:`

Reply:
Ah good catch, thanks.